Makefile

```diff
@@ -111,7 +112,7 @@ build-asan: build-debug
 build-debug: EXTRA_CMAKE_FLAGS += -DCMAKE_BUILD_TYPE=Debug -DFORCE_DEBUG_BUILD=True
 build-debug: build
 
-build-module:
+bin/cartographer-module:
```

Comment:
name changed to allow make command to look in bin/cartographer-module exists before running. This prevents duplicate runs of build-module when building appiamges